src/lib/lazy-load.js

```diff
@@ -1,18 +1,72 @@
 const lazyLoadingModifier = 'lazy-loading';
 const lazyLoadingImageClass = `n-image--${lazyLoadingModifier}`;
 const lazyLoadingWrapperClass = `n-image-wrapper--${lazyLoadingModifier}`;
+const performanceSupported = ('performance' in window && typeof window.performance.mark === 'function');
```

Comment:
`require('n-ui/utils').perfMark` does all this sort of stuff https://github.com/Financial-Times/n-ui/blob/master/utils/index.js#L59. Also hooks up to LUX -the new tool we're trialling from speedcurve